soccer/gameplay/tactics/pass_to_point_heuristic.py

```diff
@@ -0,0 +1,12 @@
+import constants
+import robocup
+import time
+
+
+class PassToPointHeuristic():
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    def calc_pass_point(self):
```

Comment:
Is there a case where this will be anything more than just a function that will be called? If not, should this just be a function that you pass into the coordinated pass?

A fun idea that I saw around is a callable. So you if change `calc_pass_point` to `__call__`, you can call the class instance like a function. Then, a function pointer, lambda function, or this heuristic class can be passed in and executed without changing any code in coordinated pass using just `self.heuristic()`. We should def pick a single way to do it, but it's a fun concept
https://en.wikipedia.org/wiki/Callable_object#In_Python